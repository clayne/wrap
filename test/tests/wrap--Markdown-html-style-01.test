wrap | /dev/null | -u | md-html-style-01.md | 0
