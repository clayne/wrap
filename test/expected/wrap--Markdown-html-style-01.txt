This is some text.  This is some more text.
<style
  type="text/css">

h1 { color : red; }

</style>
This is some text.  This is some more text.
